installer/pkg/config/libvirt/libvirt.go

```diff
@@ -31,26 +34,68 @@ type Network struct {
 }
 
 // TFVars fills in computed Terraform variables.
-func (l *Libvirt) TFVars(masterCount int) error {
+func (l *Libvirt) TFVars(masterCount int, workerCount int, etcdCount int) error {
 	_, network, err := net.ParseCIDR(l.Network.IPRange)
 	if err != nil {
 		return fmt.Errorf("failed to parse libvirt network ipRange: %v", err)
 	}
 
+	if l.BootstrapIP == "" {
+		ip, err := cidr.Host(network, 10)
+		if err != nil {
+			return fmt.Errorf("failed to generate bootstrap IP: %v", err)
+		}
+		l.BootstrapIP = ip.String()
+	}
+
 	if len(l.MasterIPs) > 0 {
 		if len(l.MasterIPs) != masterCount {
 			return fmt.Errorf("length of MasterIPs doesn't match master count")
```

Comment:
nit: this "do we already have some, and if so, do we have the right number?" check could move into `generateIPs` to save you some repetition here.